flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/LogicalTypeJsonDeserializer.java

```diff
@@ -130,6 +139,11 @@ public LogicalType deserialize(JsonNode logicalTypeNode, SerdeContext serdeCtx)
                     return deserializeArrayType(logicalTypeNode, serdeCtx);
                 case MULTISET:
                     return deserializeMultisetType(logicalTypeNode, serdeCtx);
+                case RAW:
+                    // This method only deserializes the LogicalType with `type='RAW'`,
```

Comment:
Nit. Code comment here seems useless since in `deserializeRawType` there is no fallback logic, maybe we can delete it.

Reply:
nope, the fallback means `deserializeRawType` method only handles `LogicalType with `type='RAW'`, otherwise it will be handled by `LogicalTypeParser.parse(logicalTypeNode.asText())`, see line#155